### ISSUE TITLE
MMS File Transfer/Email Option

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#25]
```
What is the expected output? 
Google Talk Client (Gtalk, Pidgin, etc) receives a file transfer request for 
MMS in the chat window.
Optionally, if this isn't possible, or as an extra option, the MMS is sent to 
an email address the gtalksms is communicating with.

What do you see instead?
Currently a disconnection occurs when I receive an MMS. I notice this when I'm 
connected to wifi. 

What version of the product are you using? On what operating system?
gtalksms 1.5 on Android 2.2 HTC Incredible / Windows 7 Ultimate with Pidgin 
client.



```

Original issue reported on code.google.com by `i.vita...@gmail.com` on 4 Jan 2011 at 9:22
- Blocked on: #80, #313

**Comment by GoogleCodeExporter** [comment]
```
For What do you see instead?
Currently a disconnection occurs when I receive an MMS. I notice this when I'm 
connected to wifi. 
==> See issue 3

For file transfer, I've to see if it's possible with our current smack library. 
I thinks we must use smackx to have extensions as file transfer.
```

Original comment by `Florent....@gmail.com` on 4 Jan 2011 at 10:26
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Was wondering if we took a look at the smackx and thought that there were ways 
to use that to make this feature possible?

Thanks by the way on all the other work you've done already!
```

Original comment by `i.vita...@gmail.com` on 2 Feb 2011 at 2:26

**Comment by GoogleCodeExporter** [comment]
```
Asmack, the current XMPP library gtalksms uses, is already capable of file 
transfer.
```

Original comment by `fschm...@gmail.com` on 2 Feb 2011 at 6:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 14 Feb 2011 at 3:12

**Comment by GoogleCodeExporter** [comment]
```
Issue 111 has been merged into this issue.
```

Original comment by `Florent....@gmail.com` on 23 Feb 2011 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
Hi,

If file transfer is hard, as I suspect it is (plus, it may not be supported by 
the receiving client ; I know mine doesn't), how about a simple notification 
that a MMS has been received? As it is, I get nothing. The xmpp message may 
have to be delayed owing to 
http://code.google.com/p/gtalksms/issues/detail?id=3.
```

Original comment by `antoine....@gmail.com` on 2 May 2012 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
Add one more for wanting a notification when an MMS is received.  If I'm 
constantly texting with GTalkSMS and someone sends an MMS during that period, 
I'd only know if I actually looked through my texts on the phone.
```

Original comment by `rod...@krweb.net` on 14 Jan 2013 at 4:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 15 Jan 2013 at 10:53
- Now blocked on: #313

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 15 Jan 2013 at 10:56
- Now blocked on: #80
